credsweeper/ml_model/ml_validator.py

```diff
@@ -132,15 +131,17 @@ def get_group_features(cls, value: str, candidates: List[Candidate]) -> Tuple[np
         return line_input, features
 
     @classmethod
-    def validate_groups(cls, group_list: List[Tuple[str, List[Candidate]]], batch_size: int) -> np.ndarray:
-        """Use ml model on list of candidate groups.
+    def validate_groups(cls, group_list: List[Tuple[str, List[Candidate]]],
+                        batch_size: int) -> Tuple[np.ndarray, np.ndarray]:
+        """Use ml model on list of candidate groups
```

Comment:
> "A docstring should be organized as a summary line (one physical line not exceeding 80 characters) terminated by a period, question mark, or exclamation point."

More details https://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
@Dmitriy-NK , please provide a linter config to check docstrings automatically in GitHub actions 
It's confusing to check it all manually